depenguinme.sh

```diff
@@ -113,6 +113,8 @@ apt-get install -y qemu
 QEMUBASE="/tmp/depenguinme"
 USENVME=0
 MYPRIMARYIP=$(ip route get 1 | awk '{print $(NF-2);exit}')
+MYSSHEXTERNALPORT="1022"
```

Comment:
Should this be somehow overrideable (either environment or preferably command line option)?